tests/test_stream.py

```diff
@@ -0,0 +1,41 @@
+from lomond.stream import WebsocketStream, FrameParser
+from lomond.errors import ProtocolError
+import pytest
+
+
+@pytest.fixture
+def stream():
+    return WebsocketStream()
+
+
+def test_constructor(stream):
+    assert isinstance(stream, WebsocketStream)
```

Comment:
To clarify, the only reason for this test was to make sure that the tests cover 100% of stream.py. However, since I created it, I also wrote other tests which also make use of the fixture, therefore this particular one does not add any significant value. I removed it